### PR TITLE
feat(config_introspect): configuration for introspection

### PR DESCRIPTION
**Pull request by deweyjose**

### CODE REVIEW

File: src/core/server.ts [hunk 1]
```diff
@@ -60,6 +61,7 @@ export class Server<C extends BaseContext> {
   mockDBConnection: boolean = false;
   resolversPath: string[];
   schema?: GraphQLSchema;
+  introspectionEnabled: boolean = true;
```
Comment (goldcaddy77) [inline src/core/server.ts]: Can we just call this `introspection`? The `Enabled` is superfluous since it's typed.

Comment (deweyjose) [reply src/core/server.ts]: sure 1 sec

File: src/core/server.ts [hunk 2]
```diff
@@ -99,6 +101,10 @@ export class Server<C extends BaseContext> {
         : process.env.NODE_ENV === 'development';
 
     this.resolversPath = this.appOptions.resolversPath || [process.cwd() + '/**/*.resolver.ts'];
+
+    if (typeof this.appOptions.introspectionEnabled !== 'undefined') {
+      this.introspectionEnabled = this.appOptions.introspectionEnabled;
+    }
```
Comment (goldcaddy77) [inline src/core/server.ts]: Since this is only being used once, we could just do `!!this.appOptions.introspection` below.

Comment (deweyjose) [reply src/core/server.ts]: Ok, didn't know about !!

**Comment by goldcaddy77** [comment]
@deweyjose did you check that `false` works as intended?

**Pull request merged**